_posts/2019-12-03-pinot-connector.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title:  "Apache Pinot Connector"
+author: Elon Azoulay
+---
+
+[Presto 328](https://prestosql.io/docs/current/release/release-328.html)
```

Comment:
It’s not clear which version will include the connector, yet, so I might just say “in an upcoming release”